### PR TITLE
full audit create a server

### PR DESCRIPTION
**Pull request by stephamon**
minor changes and updates

JIRA 1651

### CODE REVIEW

**Comment by stephamon** [comment]
Hey @cyrichardson can you take a look at this article please?

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-80b3980235/how-to/)

**Comment by cyrichardson** [comment]
On it, @stephamon.

**Comment by cyrichardson** [comment]
@stephamon Suggested changes in 
[Create a cloud server_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2077017/Create.a.cloud.server_cyrEdits.docx)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-614184f314/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-32f96fa674/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-d12dd541d5/how-to/). :bow:

File: content/cloud-servers/create-a-cloud-server.md [hunk 1]
```diff
@@ -1,32 +1,34 @@
...
+3.  In the **Server Details** section, enter a name for your server in
     the **Server Name** field.
 
 4.  From the Region list, select the region in which you want to create
     the server.
 
     For more information about regions and how to use them effectively, see [About regions](/how-to/about-regions).
 
-5.  In the **Image** section, select which operating system you want to
+5.  In the **Image** section, select which operating system that you want to
```
Comment (RackerKate) [inline content/cloud-servers/create-a-cloud-server.md]: "...select the operating system that you want to use."

File: content/cloud-servers/create-a-cloud-server.md [hunk 2]
```diff
@@ -39,31 +41,31 @@ Control Panel interface.
...
-        key will be used.
-    4.   Paste your public key into the **Public Key** field.
+    1. Click **Add Public Key** beside the SSH Key drop-down list.
+    2. Provide a Key Name for your new public key.
+    3. Paste your public key into the **Public Key** field.
 
-      **Note:** If you do not have a public key yet, click [How to get a public key](/how-to/connecting-to-a-server-using-ssh-on-linux-or-mac-os) and follow the instructions in that article. For more information on how to generate a public and private key pairs, see [Manage SSH Keypairs for Cloud Servers with-python-novaclient](/how-to/manage-ssh-key-pairs-for-cloud-servers-with-python-novaclient).
-    5.   Once you have your entered your Key Name, Region, and the Public
-        Key, click **Add Public Key**.
+       **Note:** If you do not have a public key, click [How to get a public key](/how-to/connecting-to-a-server-using-ssh-on-linux-or-mac-os) and follow the instructions in that article. For more information on how to generate a public and private key pairs, see [Manage SSH Keypairs for Cloud Servers with-python-novaclient](/how-to/manage-ssh-key-pairs-for-cloud-servers-with-python-novaclient).
```
Comment (RackerKate) [inline content/cloud-servers/create-a-cloud-server.md]: "...see [Manage SSH Keypairs for Cloud Servers with python-novaclient](...". 
I don't believe there should be a hyphen between "with" and "python-novaclient."

File: content/cloud-servers/create-a-cloud-server.md [hunk 3]
```diff
@@ -39,31 +41,31 @@ Control Panel interface.
...
 
-    <img src="{% asset_path cloud-servers/create-a-cloud-server/Screen%20Shot%202015-01-14%20at%209.30.59%20AM.png %}" width="655" height="299" />
+    <img src="{% asset_path cloud-servers/create-a-cloud-server/create-server-add-public-key.png %}" />
 
-9.  Confirm that your key is listed in the SSH Key list for your new
+    Then, confirm that your key is listed in the SSH Key list for your new
     server.
 
-10. As needed, create a new network and select the PublicNet and
+8. As needed, create a new network and select the PublicNet and
```
Comment (RackerKate) [inline content/cloud-servers/create-a-cloud-server.md]: Should "PublicNet" and "ServiceNet" be bolded?

**Pull request merged**